bundles/AdminBundle/public/js/pimcore/settings/user/user/keyBindings.js

```diff
@@ -71,7 +71,11 @@ pimcore.settings.user.user.keyBindings = Class.create({
         var generalItems = [];
 
         for (var action in mapping) {
-            if (mapping.hasOwnProperty(action)) {
+            if (mapping.hasOwnProperty(action)){
```

Comment:
what is that about?

Reply:
reverted that change 4ea76185670345f7a9d89554c20ada9d93ae16c2